Formula/gw.rb

```diff
@@ -0,0 +1,23 @@
+class Gw < Formula
+  desc "Browser for genomic sequencing data and variants"
+  homepage "https://github.com/kcleal/gw"
+  url "https://github.com/kcleal/gw/archive/refs/tags/v0.3.1.tar.gz"
+  sha256 "b479e19c2fed0704a652c0fcbd653517ce01ac429ec7753c95a583c35a733012"
+  license "MIT"
+  depends_on "fontconfig"
+  depends_on "glfw"
+  depends_on "htslib"
+  depends_on "icu4c"
+  depends_on "wget"
+  depends_on "zlib"
+
+  def install
+    system "make", "prep"
+    system "sed -i.bak 's/lglfw/lglfw3/g' Makefile && make"
+    bin.install "gw"
+  end
+
+  test do
+    system "gw", "--version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I will add this. Thanks